js/ui/templates/grade-details.es6.js

```diff
@@ -19,6 +20,14 @@ module.exports = function () {
                 ${httpsMsg(this.model.site.httpsState)}
                 <div class="float-right"></div>
             </h2>
+            <h2 class="site-info__tosdr-status">
+                ${tosdrMsg(this.model.site.tosdr)}
+                <div class="float-right"></div>
```

Comment:
What's happening with this empty div here?

Reply:
Doesn't seem to do anything but I saw the https message above has one.